src/System.Private.CoreLib/shared/System/Number.Parsing.cs

```diff
@@ -1882,15 +1886,59 @@ private static bool TrailingZeros(ReadOnlySpan<char> value, int index)
             return null;
         }
 
-        private static bool IsWhite(int ch) => ch == 0x20 || ((uint)(ch - 0x09) <= (0x0D - 0x09));
+        private static bool IsWhite(int ch) => ch == 0x20 || (uint)(ch - 0x09) <= (0x0D - 0x09) ? true : false;
```

Comment:
Is there an issue # we can include for the JIT issue that makes the `? true : false` necessary? It's counter intuitive and would be nice to clean up when the JIT issue is addressed.

Reply:
It seems to be #914, at least that's where @mikedn found the workaround.

Reply:
Yes, it's the same issue. I added comment with a link to it.